core/src/main/scala/cats/syntax/traverseFilter.scala

```diff
@@ -27,6 +27,9 @@ trait TraverseFilterSyntax extends TraverseFilter.ToTraverseFilterOps
 private[syntax] trait TraverseFilterSyntaxBinCompat0 {
   implicit def toSequenceFilterOps[F[_], G[_], A](fgoa: F[G[Option[A]]]): SequenceFilterOps[F, G, A] =
     new SequenceFilterOps(fgoa)
+
+  implicit def toTraverseCollectOps[F[_], G[_], A](fa: F[A]): TraverseCollectOps[F, G, A] =
+    new TraverseCollectOps(fa)
```

Comment:
Should this just be `TraverseFilterOps`? We can add more ops in the future; it's not specific to `traverseCollect` I think.

Reply:
I followed current implementations like `new SequenceFilterOps(fgoa)` I don't know what would be the other solution tbh

Reply:
Ah the situation for `SequenceFilter` is slighly different, since it only works for `F[G[Option[A]]]`, not `F[A]`
https://github.com/typelevel/cats/blob/0cd6d46a40143fa8593383f1b93fbb40d46191bf/core/src/main/scala/cats/syntax/traverseFilter.scala#L28

Reply:
Do you know where I could find examples to follow? I am not sure what I need to change here.

Reply:
Sure, check out any of the syntax implementations :)
https://github.com/typelevel/cats/tree/main/core/src/main/scala/cats/syntax

I'm just proposing that we rename `TraverseCollectOps` to `TraverseFilterOps`.

Reply:
I understand your point now. I thought it was a remark about implementation details. Thanks for your work on this PR !

Reply:
Ah, sorry I wasn't so clear on that. Great, thank you too!